editions/tw5.com/tiddlers/messages/WidgetMessage_ tm-modal.tid

```diff
@@ -14,6 +14,15 @@ The "currentTiddler" variable is set to the title of the modal tiddler, but can
 
 The modal message is usually generated with the ButtonWidget. The modal message is handled by the TiddlyWiki core.
 
+! The Modal Tiddler
+Within the Modal tiddler itself;
+
+*The field named "subtitle" will be used as the title of the Modal Window
+**This can contain macros, widgets and transcludions as well
```

Comment:
"transcludions" is a typo. In any case I think it's more consistent to say that the field can contain wikitext, rather than enumerating those three wikitext features